### ISSUE TITLE
Regarding adding an example on finding similar faces

### ISSUE THREAD

**Issue by sayakpaul** [keras-team/keras-io#60]
In this example, I wish to cover how to build, train and use FaceNet like models for finding similar faces. I have already prepared a [Colab Notebook](https://colab.research.google.com/drive/1WKzU-gMuW8msEuvbXcggN5TSwgl2QsBN?usp=sharing) that demonstrates an end-to-end example. It uses [`TripletSemiHardLoss`](https://www.tensorflow.org/addons/api_docs/python/tfa/losses/TripletSemiHardLoss) from TensorFlow Addons as the loss function. I used the a subset of the [LFW dataset](https://www.kaggle.com/jessicali9530/lfw-dataset).

**Comment by fchollet** [comment]
This is would be a great new example. Triplet losses are super important and we don't cover them anywhere. The notebook will need a bit of work to be turned into a proper example, but I'm confident we'll get there :)

**Comment by sayakpaul** [comment]
Sure, I will get started then :)

**Comment by sayakpaul** [comment]
@fchollet I discussed this Adrian (Rosebrock). So, we are planning on a series of blog posts on Triplet Loss and we are planning to include this example as a part of that. The series is planned for the latter half of the year (mid-July). 

 For now, I am closing this issue. We can revisit this then :)
